azkaban-common/src/main/java/azkaban/scheduler/ScheduleManager.java

```diff
@@ -112,6 +112,15 @@ public List<Schedule> getSchedules()
     return new ArrayList<>(this.scheduleIDMap.values());
   }
 
+  public List<Schedule> getALlSchedules() {
```

Comment:
ALi -> All

Reply:
Also could you please elaborate more on the difference between getSchedules and getALlSchedules? I'm curious about why we use the latter in removeScheduleOfDeletedFlows but use the former in all other places.

Reply:
They actually don't have big difference technically speaking as for the current function.
getSchedules() more focusing on maintaining its own scheduleMap (by getting updated schedules and refresh local map) and returning that map's value. It's possible that some triggers are expired/invalid, it would not be included in the scheduleMap.
getAllSchedule() is more brutal, it's directly fetching all schedules regardless the status. So if we consider deleting non-existing flows, it might be bigger set to consider.

Reply:
We observe several schedules with deleted flow in the system. That's my guess it might not include all the schedules. (Or could be not executing cleanup schedule logic in upload Project)

Reply:
Anyway I thought it's good to include a bigger set of schedules.

Reply:
Got it. Thank you so much for your clarification!